TunnelVisionLabs.ReferenceAssemblyAnnotator/TunnelVisionLabs.ReferenceAssemblyAnnotator.targets

```diff
@@ -224,79 +170,21 @@
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Linq'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Linq.Expressions'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Net.Http'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Net.Primitives'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Net.Sockets'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.ObjectModel'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Reflection'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Reflection.Extensions'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Reflection.Primitives'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Resources.ResourceManager'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Runtime'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Runtime.Extensions'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Runtime.Handles'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Runtime.InteropServices'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Runtime.InteropServices.RuntimeInformation'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Runtime.Numerics'
-                                        OR '%(Reference.NuGetPackageId)' == 'XXXSystem.Security.Cryptography.Algorithms'
-                                        OR '%(Reference.NuGetPackageId)' == 'XXXSystem.Security.Cryptography.Encoding'
-                                        OR '%(Reference.NuGetPackageId)' == 'XXXSystem.Security.Cryptography.Primitives'
-                                        OR '%(Reference.NuGetPackageId)' == 'XXXSystem.Security.Cryptography.X509Certificates'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Text.Encoding'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Text.Encoding.Extensions'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Text.RegularExpressions'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Threading'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Threading.Tasks'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Threading.Timer'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Xml.ReaderWriter'
-                                        OR '%(Reference.NuGetPackageId)' == 'System.Xml.XDocument')"
```

Comment:
😕  What property of the .NET Standard 1.x references links back to NETStandard.Library?

Reply:
I'm using `ResolvedCompileFileDefinitionsToAdd` for this, just like netcoreapp1.x already is in master.

Reply:
(thus https://github.com/tunnelvisionlabs/ReferenceAssemblyAnnotator/pull/67/commits/d13e94bd47eb2d4748f08f34cd08f952363b7ce1)